modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -32,10 +45,20 @@ spec:
     node-role.kubernetes.io/worker: ""
   ptpEventConfig:
     enableEventPublisher: true <1>
-    transportHost: amqp://<instance_name>.<namespace>.svc.cluster.local <2>
+    transportHost: <transport_host> <2>
+    storageType: "example-storage-class" <3>
 ----
 <1> Set `enableEventPublisher` to `true` to enable PTP fast event notifications.
-<2> Set `transportHost` to the AMQ router that you configured where `<instance_name>` and `<namespace>` correspond to the AMQ Interconnect router instance name and namespace, for example, `amqp://amq-interconnect.amq-interconnect.svc.cluster.local`
+<2> Set `transportHost` to the HTTP transport or AMQ router that you have configured.
+For example, for an HTTP transport, set `http://ptp-event-publisher-service-<node_name>.openshift-ptp.svc.cluster.local:9043`, where `<node_name>` is the cluster node where you have configured PTP.
+If `transportHost` is missing or empty, the default value of `http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043` is used.
+For an AMQ router, set `amqp://<instance_name>.<namespace>.svc.cluster.local` where `<instance_name>` and `<namespace>` correspond to the AMQ Interconnect router instance name and namespace.
+<3> The value of `storageType` is used to populate the `StorageClassName` field for the `PersistentVolumeClaim` (`PVC`) resource that the PTP Operator automatically deploys. The `PVC` CR resource is used to persist consumer event subscriptions.
++
+[NOTE]
+====
+If `storageType` is not set, an error occurs.
```

Comment:
the ptp daemons won't be deployed without valid storage type when event is enabled and tarnsportHopst is set to use HTTP either by  commenting TransportHost  or set to PTP service nam